themes/default/content/blog/organization-access-tokens/index.md

```diff
@@ -0,0 +1,60 @@
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - devon-grove
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+    - pulumi-enterprise
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+As enterprise adoption of the Pulumi Service has grown 350% over the last year, we've seen a strong customer demand for tools to manage automated Pulumi use cases such as CI/CD and Automation API at scale. Today we are launching Organization Access Tokens to empower our largest customers to manage automated workloads in a secure and collaborative manner.
+<!--more-->
+Enterprise and Business Critical customers of the Pulumi Service can now create Access Tokens tied to their Pulumi Organization. In addition to creating and managing Access Tokens tied to a user’s access (Personal Access Tokens), customers can utilize Organization Access Tokens, tokens which are not tied to a specific user. This feature is particularly useful for:
+
+  1. Enterprise customers using SSO/SAML/SCIM, as it enables them to create Pulumi Service tokens not tied to identities within their identity provider (IdP)
+  2. Garnering programmatic access for continuous integration and continuous delivery (CI/CD) tools and other automated processes
+
+We're excited to deliver one of the most [requested features from the community](https://github.com/pulumi/service-requests/issues/16) and a top ask from some of the largest customers using the Pulumi Service. The need to use Personal Access Tokens to manage automation and organization-specific resources can lead to some friction. When a user is deprovisioned or otherwise leaves the organization, their tokens in use no longer grant access and pipelines can become broken. Additionally, since Personal Access Tokens grant access to all resources and organizations they have access to, using personal tokens in organization automation pipelines leads to the pipeline having more privileged access than is necessary. We want to enable our customers to use the principle of least privilege. At launch, Organization Access Tokens will have write permissions to stacks in an Organization. We plan to continue to expand options for scoping this access based on feedback from our customers.
+
+See below what the experience will look like in the Pulumi Service Console:
```

Comment:
Since this is available today:

```suggestion
See below what the experience looks like in the Pulumi Service Console:
```

Reply:
@meagancojocar we dropped the Pulumi Service Console branding a bit ago, instead we should be using either web console (lowercase c and without the pulumi) or Pulumi Service UI. happy to fix this up later today if you'd like me to, let me know.